README.md

```diff
@@ -459,6 +459,21 @@ aliased to `redisConnectCallback`:
 void(const redisAsyncContext *ac, int status);
 ```
 
+If `hiredis` >= v1.1.0 is used you can alternatively set the following
+connect callback which will be passed a non-const `redisAsyncContext*` on
+invocation (e.g. allowing to set a push callback).
```

Comment:
```suggestion
Alternatively, if `hiredis` >= v1.1.0 is used, you set a connect callback
that will be passed a non-const `redisAsyncContext*` on invocation (e.g.
to be able to set a push callback on it).
```

* comma
* which vs that (look up the grammar rules if you want)
* when I read it properly, I suggest moving some words around